test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
And perhaps more importantly, a link to coveralls.io as a comment here maybe very helpful for those who aren't familiar with this stuff.